src/apps/Map/MapState.unit.test.ts

```diff
@@ -0,0 +1,115 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { rest } from 'msw';
+import {
+  LoadPlanningData,
+  loadPlanningData,
+  MapState,
+  SET_PLANNING_DATA
+} from '~/apps/Map/MapState';
+import config from '~/config';
+import { mswServer } from '../../../jest/msw/mswServer';
+import { RootState } from '~/store';
+
+describe('Plannings Map Unit tests', () => {
+  describe('loadPlanningData() thunk', () => {
+    const mockStore = configureMockStore([thunk]);
+
+    it('does not dispatch any actions if plannings data has been fetched already', async () => {
+      /* ARRANGE: mock store with initial state */
+      const preSeededMapState = {
+        planningData: true
+      } as MapState; // omitting non-relevant props
+      const preSeededRootState = {
+        MapState: preSeededMapState
+      } as RootState;
+      const store = mockStore(preSeededRootState);
+
+      /* ACT: invoke loadPlanningsThunk */
+      const loadPlanningsThunk = loadPlanningData();
+      // @ts-ignore FIXME: properly type this
+      await store.dispatch(loadPlanningsThunk);
+
+      /* ASSERT: no action should have been called */
+      expect(store.getActions()).toHaveLength(0);
+    });
+
+    it(`dispatches ${SET_PLANNING_DATA} once the plannings JSON has been fetched successfully`, async () => {
+      /* ARRANGE: mock store, intercept request and mock response */
+      const store = mockStore({
+        MapState: {} as MapState
+      });
+
+      const response = {
```

Comment:
maybe put this into a fixture json? could make the unit test more readable by shortening the file